realm/realm-library/src/testObjectServer/kotlin/io/realm/log/obfuscator/CustomFunctionObfuscatorTest.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.log.obfuscator
+
+import org.junit.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+const val CUSTOM_FUNCTION_ORIGINAL_INPUT = """{"mail":"myfakemail@mongodb.com","id":{"{$}numberInt":"666"},"options":{"device":{"appVersion":"1.0.","appId":"realm-sdk-integration-tests-grbrc","platform":"android","platformVersion":"10","sdkVersion":"10.0.0-BETA.5-SNAPSHOT"}}}"""
```

Comment:
Given that Kotlin support multiline strings when using `"""` it would be nice if this as nicely formatted with linebreaks. Makes it a lot easier to read.